site/src/examples/stepper-input/Limits.tsx

```diff
@@ -0,0 +1,32 @@
+import { FormField, FormFieldHelperText, FormFieldLabel } from "@salt-ds/core";
+import { StepperInput } from "@salt-ds/lab";
+import { useState } from "react";
+
+export const Limits = () => {
+  const [value, setValue] = useState<number | string>(2);
+  const max = 10;
+  const min = 0;
+
+  const numericValue =
+    typeof value === "number" ? value : Number.parseFloat(value);
+  const isError =
+    typeof value !== "number" || numericValue > max || numericValue < min;
+
+  return (
+    <FormField
+      validationStatus={isError ? "error" : undefined}
```

Comment:
I don't think this validation is working correctly, it triggers when I use the increment/decrement buttons

Reply:
Fixed